packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -604,17 +604,75 @@ abstract class BoxScrollView extends ScrollView {
 /// padding. To avoid this behavior, override with a zero [padding] property.
 ///
 /// {@tool sample}
+/// This example uses the default constructor for [ListView] which takes an
+/// explicit [List<Widget] of children. This [ListView]'s children are made up
+/// of [Container]s with [Text].
 ///
-/// An infinite list of children:
+/// ```dart
+/// ListView(
+///   padding: EdgeInsets.all(8.0),
+///   children: <Widget>[
+///     Container(
+///       height: 50,
+///       color: Colors.amber[600],
+///       child: Center( child: Text('Entry A')),
+///     ),
+///     Container(
+///       height: 50,
+///       color: Colors.amber[500],
+///       child: Center(child: Text('Entry B')),
+///     ),
+///     Container(
+///       height: 50,
+///       color: Colors.amber[100],
+///       child: Center(child: Text('Entry C')),
+///     ),
+///   ],
+/// )
+/// ```
+/// {@end-tool}
+/// {@tool sample}
+/// This example mirrors the previous one, creating the same list using
+/// [ListView.builder].
 ///
 /// ```dart
+/// final List entries = ['A', 'B', 'C'];
```

Comment:
const List<String> entries = <String>['A', 'B', 'C'];
const List<int> colorCodes = <int>[600, 500, 100];

Here and below

Reply:
Using `const` here and below is flagged by the analyzer, as well as if these are declared using `static const`. 
Apparently `final` is the way to go here for this template of code snippet? Or am I missing something else as well?